test/tasks/mock/MockMultisigTask.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity 0.8.15;
 import {IProxyAdmin} from "@eth-optimism-bedrock/interfaces/universal/IProxyAdmin.sol";
 import {Constants} from "@eth-optimism-bedrock/src/libraries/Constants.sol";
 import {IProxy} from "@eth-optimism-bedrock/interfaces/universal/IProxy.sol";
+import {VmSafe} from "forge-std/Vm.sol";
```

Comment:
This is messy but I want to get #624 merged and I can't commit to the branch, it need this import: https://github.com/ethereum-optimism/superchain-ops/pull/624#discussion_r1965968037